src/cache/impl/LevelDbCache.ts

```diff
@@ -94,6 +94,12 @@ export class LevelDbCache<V = any> implements SortKeyCache<V> {
     await contractCache.put(stateCacheKey.sortKey, value);
   }
 
+  async delete(contractTxId: string): Promise<void> {
+    const contractCache = this.db.sublevel<string, any>(contractTxId, { valueEncoding: 'json' });
```

Comment:
hmm..not sure how exactly this API works, but shouldn't rather sth like
`this.db.del(contractTxId)` be used here?
I.e. removing the whole contract from the 'root' database?

https://www.npmjs.com/package/level#dbdelkey-options-callback

Reply:
I tried `this.db.del(...)` and it just doesn't work. I think this is because we're using sublevels.